src/app/content/utils/archiveTreeUtils.ts

```diff
@@ -119,7 +119,7 @@ export const prevNextBookPage = (
   };
 };
 
-export const archiveTreeSectionIsBook = (section: LinkedArchiveTreeNode) => !section.parent;
+export const archiveTreeSectionIsBook = (section: LinkedArchiveTreeNode | undefined) => !section || !section.parent;
```

Comment:
This means that `undefined` is a `Book`
Shouldn't it be `=> section && !section.parent`?